geomesa-raster/LICENSE.txt

```diff
@@ -0,0 +1,176 @@
+                                 Apache License
```

Comment:
Shouldn't this file be in the main GeoMesa directory?  (And shouldn't we have already added this file to the GeoMesa source distribution?)